modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,64 @@
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the mirror registry scope of the ImageContentSourcePolicy object
+
+[role="_abstract"]
+Use the `--icsp-scope=registry` flag with the `oc adm catalog` command to widen the mirror registry scope in the ICSP file. This minimizes the number of times the cluster nodes are rebooted when a registry entry is updated.
+
+The `oc adm catalog mirror` command generates an ImageContentSourcePolicy (ICSP) object that maps the original container image repository to a new location where it is mirrored, typically inside a disconnected environment. When a new or modified ICSP is applied to a cluster, it is converted to a config file for CRI-O and placed onto each node. The process of placing the config file on a node includes rebooting that node. Having a widely scoped ICSP reduces the number of times the ICSP needs to change in the future and thus reduces the number of times a cluster must reboot its nodes.
```

Comment:
I am not particularly familiar with the specifics of the ICSPs and their effect on node reboots so I can't sign off on this bit, perhaps @kevinrizza knows more?

Reply:
Yep, this is correct

Reply:
Thanks!